uwp/Pivot-Grid/Common/Exporting.md

```diff
@@ -25,7 +25,7 @@ Refer to the following code snippet to export the SfPivotGrid to Microsoft Excel
 
 FileSavePicker savePicker = new FileSavePicker();
 savePicker.SuggestedFileName = "Sample";
-ExportPivotGridToExcel export = new ExportPivotGridToExcel(this.PivotGrid1, this.PivotGrid1.GetExportingGridStyleInfo());
+ExportPivotGridToExcel export = new ExportPivotGridToExcel(this.PivotGrid1, this.PivotGrid1.GetExportingGridStyleInfo(), "xlsx", true);
```

Comment:
@susmitha-sundar - We have 4 parameters in this method. Refer the below reference. https://github.com/essential-studio/uwp-demos/blob/6b4d2e6a66cbd4c758f1e30d3b956ee2afbcd522/PivotGrid/PivotGrid/View/Exporting.xaml.cs#L253